Dockerfile.ocim

```diff
@@ -0,0 +1,24 @@
+FROM python:3.6
+
+WORKDIR /usr/src/ocim
```

Comment:
This is totally fine, though I personally prefer shorter paths like `/ocim` for these kind of dev containers.